css/style.css

```diff
@@ -0,0 +1,346 @@
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 100%;
+    color:white;
+}
+
+.contact-us div div{
+     font-size: 20px;
+     margin-left:30%;
+     margin-right:30%;
+}
+
+.contact--content{
+    margin-bottom:5%;
+}
+
+.btn-primary--bg{
+    margin-left: auto;
+    margin-right: auto;
+}
+
+nav .menu{
+    display:none;
+    background-color:transparent;
+    border:none;
+}
+.nav{
+    display:none;
+    font-size:20px; 
+}
+
+.nav a{
+    display:block;
+    background-color:white;
+    color:#00458A !important;
+    padding:2%;
+}
+
+
+@media only screen and (max-width: 768px){
+
+    .content-top{
+        max-width: 768px;
+        display: flex;
+        flex-direction:column-reverse;
+        justify-content: center;
+        padding:9%;
+
+       
+    }
+    
+    .content-top div{
+        width: 100%;
+        max-width: 680px;
+    }
+    
+    .content-top div div{
+        width: 100%;
+        padding-right: 0;
+    }
+
+       
+    .content-top img{
+        width: 100%;
+        height: 100%;
+        max-width:680px;
+        margin-bottom: 10%;
+    }
+    .content-top .hero--title{
+        margin-bottom:10%;
+    }
+    .hero--title{
+        font-size: 2.5rem;
+    }
+
+    .paragraph{
+        font-size: 1.7rem;
+        line-height: 2rem;
+    }
+ 
+    
+    .content-middle{
+        max-width: 768px;
+        padding:9%;
+        display: block;
+        justify-content: center;
+        padding-bottom: 10%;
+    }
+    
+    .content-middle div{
+        width: 100%;
+        padding-bottom: 5%;
+        max-width:680px;
+    }
+    
+    .content-middle div div{
+        width: 100%;
+
+    }
+    .content-middle img{
+        width: 100%;
+        height: 100%;
+        max-width:680px;
+    }
+    
+    .researching{
+        text-align: left;
+        padding:0;
+        padding-bottom: 5%;
+    }
+    
+    .researchFocus{
+        max-width: 768px;
+        background-image: url("../images/bulbs.png");
+        padding: 8%;
+        background-position: bottom;    
+        background-repeat: no-repeat;
+        background-size: 20%;
+        background-color:  #F8F9FB;
+        padding-bottom:8%;
+    }
+    
+    .researchContent{
+  
+        display: block;
+        justify-content: space-evenly;
+    }
+    
+    .card{
+        margin:0%;
+        margin-bottom:5%;
+        background-color: white;
+        max-width:680px;
+    }
+
+    .card .paragraph{
+        margin:5%;
+    }
+
+    .card .header{
+        font-size:1.5rem;
+        line-height: 30px;
+        padding:8%;
+    }
+    
+    .content-bottom{
+        max-width: 768px;
+        display:block;
+    }
+    
+    .content-bottom .bottom{
+        text-align: center;
+        width:100%;
+    }
+    .content-bottom p{
+        min-height:0px;
+        margin:7%;
+    }
+    .content-bottom img{
```

Comment:
here needs to add width:100% for responsive.
![127 0 0 1_5500_index html(iPhone 6_7_8 Plus)](https://user-images.githubusercontent.com/36991424/71319475-cd5abb00-24d9-11ea-924b-41c92ecdfd2b.png)